frontend/.storybook/main.ts

```diff
@@ -1,5 +1,7 @@
 import type { StorybookConfig } from "@storybook/react-webpack5";
 
+const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin");
```

Comment:
여기 왜 ESM 모듈이 아니라 CJS 모듈 방식을 이용했나요?

```suggestion
import TsconfigPathsPlugin from "tsconfig-paths-webpack-plugin";
```

Reply:
다른 코드 복붙했습니다.. 수정했습니다 감사합니다!!